helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
       containers:
       - name: {{ template "iqserver.fullname" . }}
         image: {{ .Values.iq.imageName }}
+        env:
+          - name: SONATYPE_INTERNAL_HOST_SYSTEM
+            value: Operator
```

Comment:
this should look familiar from the PR on the helm charts, but it's a different, older copy.